README.md

```diff
@@ -33,6 +34,28 @@ This template comes with:
 1. `rails s`
 1. You can now try your app!
 
+## How to use with docker
+
+1. Have `docker` and `docker-compose` installed (You can check this by doing `docker -v` and `docker-compose -v`)
+2. Modify the following lines in the `database.yml` file:
+  ``` yaml
+  default: &default
+    adapter: postgresql
+    encoding: unicode
+    pool: 5
+    username: postgres
+    password: postgres
+    host: db
```

Comment:
Since `database.yml` is committed into the repo I think we can use env vars and not make the user configure stuff. Like `host: <%= ENV.fetch('DB_HOST') { 'localhost' } %>

Reply:
Actually if we move to dotenv, then I could pick the en variables from there to use them in here and pass them from docker to rails. I'll wait until that pr is made/merged and update accordingly. Wdyt?

Reply:
+1 to that, still needs to be on the guide though right? To Docker or not to Docker